[2023-06-27T15:09:08.376+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=manual__2023-06-27T15:09:05.719299+00:00/task_id=get_choice permission to 509
[2023-06-27T15:09:08.377+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=manual__2023-06-27T15:09:05.719299+00:00/task_id=get_choice permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=manual__2023-06-27T15:09:05.719299+00:00/task_id=get_choice'
[2023-06-27T15:09:08.406+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=manual__2023-06-27T15:09:05.719299+00:00/task_id=get_choice permission to 509
[2023-06-27T15:09:08.407+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=manual__2023-06-27T15:09:05.719299+00:00/task_id=get_choice permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=manual__2023-06-27T15:09:05.719299+00:00/task_id=get_choice'
[2023-06-27T15:09:08.565+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 2_Test_Badge_5_Loop.get_choice manual__2023-06-27T15:09:05.719299+00:00 [queued]>
[2023-06-27T15:09:08.573+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 2_Test_Badge_5_Loop.get_choice manual__2023-06-27T15:09:05.719299+00:00 [queued]>
[2023-06-27T15:09:08.574+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-06-27T15:09:08.590+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): get_choice> on 2023-06-27 15:09:05.719299+00:00
[2023-06-27T15:09:08.595+0000] {standard_task_runner.py:57} INFO - Started process 2694 to run task
[2023-06-27T15:09:08.601+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '2_Test_Badge_5_Loop', 'get_choice', 'manual__2023-06-27T15:09:05.719299+00:00', '--job-id', '415', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpv84g1pk9']
[2023-06-27T15:09:08.603+0000] {standard_task_runner.py:85} INFO - Job 415: Subtask get_choice
[2023-06-27T15:09:08.656+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=manual__2023-06-27T15:09:05.719299+00:00/task_id=get_choice permission to 509
[2023-06-27T15:09:08.657+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=manual__2023-06-27T15:09:05.719299+00:00/task_id=get_choice permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=manual__2023-06-27T15:09:05.719299+00:00/task_id=get_choice'
[2023-06-27T15:09:08.659+0000] {task_command.py:410} INFO - Running <TaskInstance: 2_Test_Badge_5_Loop.get_choice manual__2023-06-27T15:09:05.719299+00:00 [running]> on host 065d269f5e6c
[2023-06-27T15:09:08.791+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='2_Test_Badge_5_Loop' AIRFLOW_CTX_TASK_ID='get_choice' AIRFLOW_CTX_EXECUTION_DATE='2023-06-27T15:09:05.719299+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-27T15:09:05.719299+00:00'
[2023-06-27T15:09:08.793+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-27T15:09:08.794+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /dbt && dbt run-operation get_stream_value --args \'{                            "stream_name": "ed_cdc_stream",                             "dry_run": false}\'']
[2023-06-27T15:09:08.805+0000] {subprocess.py:86} INFO - Output:
[2023-06-27T15:09:11.871+0000] {subprocess.py:93} INFO - [0m15:09:11  Running with dbt=1.5.1
[2023-06-27T15:09:19.528+0000] {subprocess.py:93} INFO - [0m15:09:19  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2023-06-27T15:09:19.529+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2023-06-27T15:09:19.530+0000] {subprocess.py:93} INFO - - models.badge_5.time_of_day_lu
[2023-06-27T15:09:19.594+0000] {subprocess.py:93} INFO - [0m15:09:19  Found 3 models, 0 tests, 0 snapshots, 0 analyses, 456 macros, 0 operations, 1 seed file, 0 sources, 0 exposures, 0 metrics, 0 groups
[2023-06-27T15:09:20.950+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-06-27T15:09:21.008+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=2_Test_Badge_5_Loop, task_id=get_choice, execution_date=20230627T150905, start_date=20230627T150908, end_date=20230627T150921
[2023-06-27T15:09:21.051+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-27T15:09:21.090+0000] {taskinstance.py:2651} INFO - 1 downstream tasks scheduled from follow-on schedule check
